frontend/src/tests/offer-tests.js

```diff
@@ -74,9 +74,30 @@ export function offersTests(api) {
         expect(newOffer.emailed_date).not.toBeNull();
     });
 
-    it.todo(
-        "when an offer is emailed a second time, the emailed date is updated to the most recent emailed date"
-    );
+    it("email the active offer again", async () => {
+        // record previoud emailed_date
```

Comment:
You should install a spell checker that spell-checks comments to catch things like `previoud`.